src/network/websocket/WebSocketServer.cpp

```diff
@@ -94,9 +94,9 @@ bool SingleClientWSServer::addProtocol(std::unique_ptr<WebSocketProtocol> protoc
 void SingleClientWSServer::sendRawString(const std::string& protocolPath,
 										 const std::string& str) {
 	std::lock_guard lock(protocolMapMutex);
-	auto entry = protocolMap.find(protocolPath);
-	if (entry != protocolMap.end()) {
-		auto& protocolData = entry->second;
+	auto protocolDataOpt = this->getProtocol(protocolPath);
+	if (protocolDataOpt.has_value()) {
+		auto& protocolData = protocolDataOpt.value().get();
```

Comment:
This is super minor, but imo chaining getters is a little ugly. We could instead do:
```c++
ProtocolData& protocolData = protocolDataOpt.value();
```
Could you replace that everywhere?